src/commands/runTestCommand.ts

```diff
@@ -1,9 +1,17 @@
+import { relative } from "path";
+import { WorkspaceFolder } from "vscode";
+
 import { getTestRunner } from "../runners/TestRunnerFactory";
 
-async function runTest(rootPath, fileName, testName) {
+async function runTest(
+  rootPath: WorkspaceFolder,
```

Comment:
rootPath is a string

![image](https://user-images.githubusercontent.com/41467575/43043785-989e06e2-8d9a-11e8-86a0-f2c55c02dfe4.png)